packages/circuit-ui/components/Headline/Headline.docs.mdx

```diff
@@ -4,25 +4,118 @@ import { Status, Props, Story } from '../../../../.storybook/components';
 
 <Status.Stable />
 
-Headlines are used for titling major sections of the interface. They help better organize the information for our users providing useful labels for each possible section.
+Headlines are used for titling sections of the interface. They're an essential
+part of a page's structure, helping organize information for our users.
 
 <Story id="typography-headline--base" />
 <Props />
 
+## Component variations
+
+### Sizes
+
+The Headline component comes in four sizes.
+
+Use the `four` size for card headers and `three` for page titles in web
+applications. For specific use cases such as landing pages, consider using the
+[Title](Typography/Title) component.
+
+<Story id="typography-headline--sizes" />
+
 ## Accessibility
 
-Nest headings by their rank (or level). The most important heading has rank 1 (`h1`), the least important heading rank 6 (`h6`). Headings with an equal or higher rank start a new section, headings with a lower rank start new subsections that are part of the higher-ranked section.
+Headings are a fundamental aspect of creating accessible interfaces. Sighted
+users rely on them to easily find what they want on a page, while assistive
+technology allows users to jump between them for easier access to the page's
+different sections.
 
-Skipping heading ranks can be confusing and should be avoided where possible: Ensure that an `h2` is not followed directly by an `h4`, for example. It is ok to skip ranks when closing subsections, for instance, an `h2` beginning a new section can follow an `h4` as it closes the previous section.
+### Best practices
 
-[Learn more about a proper page structure](https://www.w3.org/WAI/tutorials/page-structure/headings/).
+#### Each page should have a single `<h1>`
 
-## Component variations
+The `<h1>` represents the page title. Our pages should all have one (and only
```

Comment:
I would avoid using "our", "we", or "us" when writing docs. At best, it's ambiguous (who's "we"?), at worst, it's exclusionary (I don't belong to "us"). 

```suggestion
The `<h1>` represents the page title. Pages should all have one (and only
```